Add information about Terraform version and k9s tool for debugging (#354)
Quansight/qhub-cloud: docs/source/06_developers_contrib_guide/04_local_development.md

  The 'metallb' addon is enabled
```

### Debug your Kubernets cluster 

>  K9s is a terminal-based UI to manage Kubernetes clusters that aims to 

Reviewer: ```suggestion
 [K9s](https://k9scli.io/) is a terminal-based UI to manage Kubernetes clusters that aims to 
```

 simplify navigating, observing, and managing your applications in K8s. 
 K9s continuously monitors Kubernetes clusters for changes and provides 
 shortcut commands to interact with the observed resources becoming a 
 fast way to review and resolve day-to-day issues in Kubernetes. It's 

>  defiitly a huge improvement to the general workflow and a best-t-have 

Reviewer: ```suggestion
 definitely a huge improvement to the general workflow and a best-to-have 
```

 tool for debugging yout Kubernets cluster sessions. 

Installation can be done on a Mac, in Windows, and Linux. Instructions 
for each operating system can be found [here](https://github.com/derailed/k9s). 
Be sure to complete installation to be able to follow along.

By default, K9s starts with the standard directory that is set as the 
context (in this case minkube). To view all the current process press `0`:

![Image of K9s termina UI](image_here)

---
**NOTE**
  In some circumstances you will be confronted with the 
  necessity to inspect in your ‘localhost’ any services launched by 
  your cluster. For instance, if your cluster has some problem with the 
  network traffic tunnel configuration it may limit or block the user's 
  access to destination resources over the connection. 

> K9s port-forward option `<shift-f>` allows you to access and interact 

Reviewer: ```suggestion
K9s port-forward option `<kbd>shift</kbd> + <kbd>f</kbd>` allows you to access and interact 
```
